source/geo.txt

```diff
@@ -0,0 +1,303 @@
+  the MongoDB Headquarters:
+
+  .. code-block:: python
+
+     {
+         "type": "Point",
+         "coordinates": [-73.856077, 40.848447] 
+     }
+
+- ``LineString``: an array of two or more positions that forms a series of line
+  segments. A ``LineString`` can represent a path, route, border, or any other linear
+  geospatial data. The following ``LineString`` represents a segment of
+  the Great Wall of China:
+
+  .. code-block:: python
+
+     {
+         "type": "LineString",
+         "coordinates": 
+         [[116.572, 40.430],
+         [116.570, 40.434],
+         [116.567, 40.436],
+         [116.566, 40.441]]
+     }
+
+- ``Polygon``: an array of positions in which the first and last
+  position are the same and enclose some space. The following
+  ``Polygon`` roughly represents the land within the Vatican City:
+
+  .. code-block:: python
+
+     {
+         "type": "Polygon",
+         "coordinates": 
+         [[[12.446086, 41.901977],
+         [12.457952, 41.901559],
+         [12.455375, 41.907351],
+         [12.449863, 41.905186],
+         [12.446086, 41.901977]]]
+     }
+
+To learn more about the GeoJSON types you can use in MongoDB, see the
+:manual:`GeoJSON manual entry </reference/geojson/>`.
+
+For more information on the GeoJSON format, see the
+`official IETF specification <https://datatracker.ietf.org/doc/html/rfc7946>`__.
+
+Legacy Coordinate Pairs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Use legacy coordinate pairs to store data that represents geospatial information
+on a two-dimensional plane.
+
+Legacy coordinate pairs are represented by an array of two values, in which the first value
+represents the ``x`` axis value and the second represents the ``y`` axis value.
+
+For more information on legacy coordinate pairs, see the
+:manual:`MongoDB server manual page on legacy coordinate pairs </geospatial-queries/#legacy-coordinate-pairs>`.
+
+Geospatial Indexes
+------------------
+
+To enable querying on geospatial data, you must create an index that
+corresponds to the data format. The following index types enable geospatial
+queries:
+
+- ``2dsphere``, used for GeoJSON data
+- ``2d``, used for legacy coordinate pairs
+
+To learn more about how to create geospatial indexes, see the :ref:`geospatial-indexes`
+section of the Indexes guide.
+
+Query Operators
+~~~~~~~~~~~~~~~
+
+To query geospatial data, use one of the following query operators:
+
+- ``$near``
+- ``$geoWithin``
+- ``$nearSphere``
+- ``$geoIntersects`` (*requires a 2dsphere index*)
+
+When using the ``$near`` operator, you can specify the following distance operators:
+
+- ``$minDistance``
+- ``$maxDistance``
+
+When using the ``$geoWithin`` operator, you can specify the following shape operators:
+
+- ``$box``
+- ``$polygon``
+- ``$center``
+- ``$centerSphere``
+
+For more information on geospatial query operators, see
+:manual:`Geospatial Query Operators </geospatial-queries/#geospatial-query-operators>` in
+the server manual.
```

Comment:
What could be confusing here is understanding which ones can be used in `find` and `aggregate` queries. I know in the geospatial querying it goes deeper in, but what are thoughts on specifying which one is only available in `aggregate`.